core/tests/protractor_desktop/navigation.js

```diff
@@ -213,7 +213,12 @@ describe('Static Pages Tour', function() {
     expect(element(
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
-
+  it('visits the Welcome Guide Page', function() {
```

Comment:
Have newlines between suites to enhance readability. So don't delete the one above, and add one after this it block.

Reply:
done!